packages/@aws-cdk/aws-msk/test/cluster.test.ts

```diff
@@ -102,18 +102,61 @@ describe('MSK Cluster', () => {
       });
 
       test('fails if tls encryption is set to tls and plaintext', () => {
-        expect(() => new msk.Cluster(stack, 'Cluster', {
-          clusterName: 'cluster',
-          kafkaVersion: msk.KafkaVersion.V2_6_1,
-          vpc,
-          encryptionInTransit: {
-            clientBroker: msk.ClientBrokerEncryption.TLS_PLAINTEXT,
-          },
-          clientAuthentication: msk.ClientAuthentication.sasl({
-            scram: true,
-          }),
-        })).toThrow(
-          'To enable SASL/SCRAM authentication, you must only allow TLS-encrypted traffic between clients and brokers.',
+        expect(
```

Comment:
What about a positive test? to make sure the CloudFormation template contains the right config?

Reply:
Sorry, I was a bit eager requesting a review. This probably should've been a draft, but I'm getting back to it today.